src/main/java/com/endava/endabank/dto/user/UpdatePasswordDto.java

```diff
@@ -0,0 +1,27 @@
+package com.endava.endabank.dto.user;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class UpdatePasswordDto {
+    private String token;
+    private String oldPassword;
+    @Size(min = 8, max = 20)
+    @Pattern(regexp = "((?=.*\\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[\\W]).{8,20})",
+            message = "1 Capital, 1 Special character, 1 Number and 8 to 20 digits,")
```

Comment:
Hi,

These messages should be located in a properties file.

You should have the following:

inside the resources folder a properties file "messages.properties" and the properties like this:
validation.eventDto.isDeleted.notNull= IsDeleted can't be  null.

inside your main class the following beans:
`	@Bean
	public MessageSource messageSource() {
		ReloadableResourceBundleMessageSource messageSource = new ReloadableResourceBundleMessageSource();
		messageSource.setBasename("classpath:messages/messages");
		messageSource.setDefaultEncoding("UTF-8");
		return messageSource;
	}
	
	@Bean
	public LocalValidatorFactoryBean validator() {
		LocalValidatorFactoryBean bean = new LocalValidatorFactoryBean();
		bean.setValidationMessageSource(messageSource());
		return bean;
	}`

inside the dto class, something like this:
@Size(max = 1024, message = "{validation.eventDto.description.size}")